docs/topics/dokka-gradle-configuration-options.md

```diff
@@ -0,0 +1,871 @@
+[//]: # (title: Dokka Gradle configuration options)
+
+Dokka has many configuration options to tailor your and your reader's experience.
```

Comment:
```suggestion
Dokka has many configuration options to customize your and your reader's experience.
```

This meaning of "tailor" usually is used with something we're tailoring stuff _to_.